policyengine_us/variables/gov/states/oh/tax/income/credits/lump_sum_distribution/oh_distribution_credit.py

```diff
@@ -0,0 +1,13 @@
+from policyengine_us.model_api import *
+
+
+class oh_distribution_credit(Variable):
```

Comment:
```suggestion
class oh_lump_sum_distribution_credit(Variable):
```

Reply:
right? of so, same for eligible and person 

adjust the .py and yaml file names